docker_db_config_template/create_certificate.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
I think, we can integrate create_certificate.sh directly into the python package and don't need to copy it, as we do it for the EXAConf templates. We need to copy the templates, because we use symlinks which are not supported in python packages.

Reply:
Done

Reply:
Use a different directory

Reply:
Done